core/examples/op2.rs

```diff
@@ -44,10 +44,7 @@ fn main() -> Result<(), Error> {
 
   let future = async move {
     let mod_id = js_runtime
-      .load_main_module(
-        &main_module,
-        Some(deno_core::FastString::from(module_code)),
-      )
+      .load_main_module(&main_module, module_code)
```

Comment:
Nitpick - I'd prefer to make `load_main_module` work on URL and `load_main_module_from_code` to accept a second param. TIOLI